code/HUXt.py

```diff
@@ -1202,36 +1340,35 @@ def huxt_constants():
     Return some constants used in all HUXt model classes
     """
     twopi = 2.0*np.pi
-    daysec = 24*60*60
+    daysec = 24*60*60 *u.s
     kms = u.km / u.s
     alpha = 0.15*u.dimensionless_unscaled  # Scale parameter for residual SW acceleration
     r_accel = 50*u.solRad  # Spatial scale parameter for residual SW acceleration
-    synodic_period = 27.2753*daysec*u.s  # Solar Synodic rotation period from Earth.
+    synodic_period = 27.2753*daysec  # Solar Synodic rotation period from Earth.
     v_max = 2000*kms
+    buffertime=5*u.day
     constants = {'twopi': twopi, 'daysec': daysec, 'kms': kms, 'alpha': alpha,
-                 'r_accel': r_accel, 'synodic_period': synodic_period, 'v_max': v_max}
+                 'r_accel': r_accel, 'synodic_period': synodic_period, 'v_max': v_max,
+                 'buffertime': buffertime}
     return constants
 
 
-def radial_grid():
+def radial_grid(r_min = 30.0 *u.solRad, r_max = 240.0*u.solRad, Nr = 140):
     """
     Define the radial grid of the HUXt model (1D and 2D)
     """
-    r_min = 30.0
-    r_max = 240.0
-    Nr = 140
+     
     r, dr = np.linspace(r_min, r_max, Nr, retstep=True)
-    r = r*u.solRad
-    dr = dr*u.solRad
+    #r = r*u.solRad
+    # dr = dr*u.solRad
     rrel = r - r[0]
     return r, dr, rrel, Nr
 
 
-def longitude_grid():
+def longitude_grid(Nlon = 128):
```

Comment:
Similar to the radial_grid comment about adding checks for a senseible number of longitude points?